pz_api/models.py

```diff
@@ -1,5 +1,209 @@
 from __future__ import unicode_literals
```

Comment:
Sobre lo que comenta @Virako [aquí](https://github.com/huelva-developers/pz-api/pull/1#issuecomment-274645863), esta linea según comentan [aquí](http://python-future.org/unicode_literals.html) es para cuando tienes código de python2 y exportas a python3.
Por otro lado, comentar que tanto en este archivo como en el resto, lo ha creado automáticamente al crear el proyecto Django.
La pregunta es, lo dejo? lo borro? que opináis...